spotbugs/src/main/java/edu/umd/cs/findbugs/detect/RuntimeExceptionCapture.java

```diff
@@ -200,8 +216,20 @@ public void visit(CodeException obj) {
                             System.out.println("Dead exception store at " + first);
                         }
                         caughtException.dead = true;
-                        break;
                     }
+
+                    // See if the excpetion is rethrown at the end of the handler.
```

Comment:
Misspelled "exception"

Reply:
This conversation can be resolved, since got fixed in https://github.com/spotbugs/spotbugs/pull/2112/commits/7b0d8382230ded583db874c4f0f0900a5f8505c8.